.yarnrc.yml

```diff
@@ -0,0 +1,16 @@
+compressionLevel: 0
+
+enableScripts: false
+
+enableGlobalCache: true
+
+# 1 week
+npmMinimalAgeGate: 10080
+
+npmPreapprovedPackages:
+  - '@electron/*'
+  - 'lerna/*'
```

Comment:
Did we want to allowlist Lerna packages? If so, this should just be `lerna`, but I also see that you expected this check to fail so should we just remove it from this array?